velox/common/caching/AsyncDataCache.cpp

```diff
@@ -661,26 +660,26 @@ void AsyncDataCache::backoff(int32_t counter) {
 
 void AsyncDataCache::incrementNew(uint64_t size) {
   newBytes_ += size;
-  if (!ssdCache_) {
+  if (ssdCache_ == nullptr) {
     return;
   }
   if (newBytes_ > nextSsdScoreSize_) {
     // Check next time after replacing half the cache.
     nextSsdScoreSize_ = newBytes_ +
-        std::max<int64_t>(cachedPages_ * memory::AllocationTraits::kPageSize,
+        std::max<int64_t>(memory::AllocationTraits::pageBytes(cachedPages_),
                           1UL << 28);
     ssdCache_->groupStats().updateSsdFilter(ssdCache_->maxBytes() * 0.9);
   }
 }
 
 void AsyncDataCache::possibleSsdSave(uint64_t bytes) {
   constexpr int32_t kMinSavePages = 4096; // Save at least 16MB at a time.
-  if (!ssdCache_) {
+  if (ssdCache_ == nullptr) {
     return;
   }
 
   ssdSaveable_ += bytes;
-  if (ssdSaveable_ / memory::AllocationTraits::kPageSize >
+  if (memory::AllocationTraits::numPages(ssdSaveable_) >
```

Comment:
This will be a round up, whereas before there was round down. Shouldn't matter much though as RHS is very big